pkg/virt-handler/vm.go

```diff
@@ -34,6 +34,7 @@ import (
 	"strings"
 	"time"
 
+	pkgerrors "github.com/pkg/errors"
```

Comment:
Why not std `errors`?

Reply:
Maybe  I wasn't clear. I was asking for `"errors"` replace by `"github.com/pkg/errors"`

Reply:
I see. Yes, it can be replaced by the std `errors`. Done.